code/modules/food_and_drinks/kitchen_machinery/cooker.dm

```diff
@@ -2,11 +2,11 @@
 	name = "cooker"
 	desc = "You shouldn't be seeing this!"
 	layer = 2.9
-	density = 1
-	anchored = 1
+	density = TRUE
+	anchored = TRUE
 	use_power = IDLE_POWER_USE
 	idle_power_usage = 5
-	var/on = 0
+	var/on = FALSE
 	var/onicon = null
 	var/officon = null
 	var/openicon = null
```

Comment:
cannot comment on it, but theres a couple more bools bellow this point.  `upgradeable`, `has_specials`, `burns`